src/views/empty-state.component.js

```diff
@@ -11,6 +11,14 @@
  *   <li>.icon   - (string) class for main icon. Ex. 'pficon pficon-add-circle-o'
  *   <li>.title  - (string) Text for the main title
  *   <li>.info  - (string) Text for the main informational paragraph
+ *   <li>.helpLink - (object) Contains url specific properties and actions
+ *   <ul style='list-style-type: none'>
+ *     <li>.label - (string) Text label which appears before the
+ *     <li>.urlLabel - (string) Text for the clickable portion of the link
+ *     <li>.url - (string) Text for url path
```

Comment:
This is optional, right?

Reply:
Yes this is optional. Will make sure to note that